DailyScreenshot/DailyScreenshot.csproj

```diff
@@ -47,6 +47,7 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="dependencys\GenericModConfigMenuAPI.cs" />
```

Comment:
Nit: Typo, dependencies. Not worth requesting a change (unnecessary delay), can fix while working on other changes before releasing next update.